assets/stylesheets/main.css

```diff
@@ -195,10 +187,20 @@ a:hover#join {
 }
 
 .bg {
-    height: calc(100vh - 45px);   /* For all sections except #1, heigh does not include the height of WCG header. Refer to #section1 below */
+    height: 100vh;   /* For all sections except #1, height does not include the height of WCG header. Refer to #section1 below */
```

Comment:
Decided to leave bg as-is (i.e. height: 100vh;), and make custom classes for sections (e.g. bgSec3)